stable/elasticsearch/templates/configmap.yaml

```diff
@@ -39,13 +39,11 @@ data:
         ping.unicast.hosts: ${DISCOVERY_SERVICE:}
         minimum_master_nodes: ${MINIMUM_MASTER_NODES:2}
 
-{{- if .Values.cluster.xpackEnable }}
```

Comment:
How does this fix the problem?

Reply:
By simply removing the if statement, we force the xpack configuration to either the env vars or the defaults, which fixes this behavior from ES5.

So instead of using the default values from 5.x.x (which are true), we use the env vars `${XPACK_XX_YY:false}`. This means that, by default, elasticsearch installed using this chart won't have xpack enabled, which fixes the issue with the k8s probes.

Reply:
I remember having the same problem on non-oss ES5 docker image, I had to explicitely set xpack.monitoring.enabled and xpack.security.enabled to false.

Reply:
If we remove the statement the default image will not boot because it doesn't have xpack.
Which image are you using?

Reply:
I'm using `docker.elastic.co/elasticsearch/elasticsearch`, I'll try with the default again, if that doesn't work I'll rethink the solution and come up with something else.

Reply:
The default in the chart is the `elasticsearch-oss` container which does not have xpack so this will break clusters using that.
If you aren't using the default image you need to maintain the xpack setup yourself.